.github/workflows/build-in-devcontainer.yaml

```diff
@@ -88,18 +89,23 @@ jobs:
             AWS_SECRET_ACCESS_KEY=${{ env.AWS_SECRET_ACCESS_KEY }}
           runCmd: |
             set -e;
-
             mkdir -p ~/.config/pip/;
             cat <<EOF >> ~/.config/pip/pip.conf
             [global]
             extra-index-url = https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
             EOF
 
-            rapids-make-${PYTHON_PACKAGE_MANAGER}-env || true;
+            rapids-make-${PYTHON_PACKAGE_MANAGER}-env;
 
-            if ! grep -qE "^$" <<< "${{ inputs.extra-repo-deploy-key }}"; then
+            if test -n "${{ inputs.extra-repo-deploy-key }}" \
+            || test -n "${{ inputs.extra-repo-deploy-key-2 }}"; then
               if ! pgrep ssh-agent >/dev/null 2>&1; then eval "$(ssh-agent -s)"; fi;
-              ssh-add - <<< "${{ secrets[inputs.extra-repo-deploy-key] }}";
+              if test -n "${{ inputs.extra-repo-deploy-key }}"; then
+                ssh-add - <<< "${{ secrets[inputs.extra-repo-deploy-key] }}";
+              fi
+              if test -n "${{ inputs.extra-repo-deploy-key-2 }}"; then
+                ssh-add - <<< "${{ secrets[inputs.extra-repo-deploy-key-2] }}";
+              fi
```

Comment:
That applies to untrusted inputs. These inputs are fixed and controlled by us. In the worst case if a user passes a bad value as `inputs.extra-repo-deploy-key`, these lines will evaluate to an empty string since their malicious string wouldn't exist as a key in the `secrets` context.

Reply:
I updated the logic to test against `'${{ secrets[inputs.extra-repo-deploy-key] }}'` everywhere. This guarantees if a user is somehow able to change `inputs.extra-repo-deploy-key`, it only gets passed to `ssh-add` if it's a key in the `secrets` context.